end2end-data/firebase-export-metadata.json

```diff
@@ -1,12 +1,12 @@
 {
-  "version": "13.27.0",
+  "version": "13.23.1",
   "firestore": {
     "version": "1.19.8",
     "path": "firestore_export",
     "metadata_file": "firestore_export/firestore_export.overall_export_metadata"
   },
   "auth": {
-    "version": "13.27.0",
+    "version": "13.23.1",
```

Comment:
I don't see any issues with this. However, could you clarify why we downgrade those packages?

Reply:
I was probably using an older version of `firebase-cli`.
This changed the version because I needed to change the emulator data since some fields were missing in the original one 😅

But I think this doesn't impact at all the project nor the e2e-tests, but @ramshty can confirm that :)

Reply:
Okay, yess I don't think this will impact the project, I was just wondering if there was a reason for it. For me we can merge this without problem !